### PR TITLE
feat: add basic-tooling schematic

### PR DESCRIPTION
**Pull request by neodmy**
## Related Issue

#45 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by inigomarquinez — APPROVED** [review]
LGTM!

**Review by jonatan-martinez — APPROVED** [review]

**Review by nanotower — COMMENTED** [review]
👌

File: packages/@guidesmiths/cuckoojs-schematics/src/basic-tooling/README.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ package:
...
 
 ## Options
 
-| Option     | Description                                                         | Requiered | Type | Default    |
-|------------|---------------------------------------------------------------------|---|---|------------|
-| `directory` | Root folder of your NestJS project                                  | false | string | `.`        |
+| Option        | Description                                                         | Requiered | Type | Default    |
+|---------------|---------------------------------------------------------------------|---|---|------------|
+| `directory`   | Root folder of your NestJS project                                  | false | string | `.`        |
+| `skipInstall` | Root folder of your NestJS project                                  | false | string | `.`        |
```
Comment (nanotower) [inline packages/@guidesmiths/cuckoojs-schematics/src/basic-tooling/README.md:20]: Is the default value wrong? Shouldn't default as false?

**Review by nanotower — APPROVED** [review]
👍🏻

File: packages/@guidesmiths/cuckoojs-schematics/src/basic-tooling/basic-tooling.factory.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import {NodePackageInstallTask} from '@angular-devkit/schematics/tasks'
+
+interface Options {
+  directory: string;
+  skipInstall: boolean;
+}
+
+export function main(options: Options): Rule {
+	return (tree: Tree, context: SchematicContext) => {
+		context.logger.info('Adding Husky, Commitlint, Gitignore, Nvmrc...');
```
Comment (inigomarquinez) [inline packages/@guidesmiths/cuckoojs-schematics/src/basic-tooling/basic-tooling.factory.ts:21]: Sorry for my TOC, but I think this tools should start by lowercase 😬

**Review by inigomarquinez — CHANGES_REQUESTED** [review]
Just one comment

**Review by inigomarquinez — APPROVED** [review]

**Pull request merged**